menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -96,8 +95,11 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
   @Value("${menas.oozie.menasApiURL:}")
   val menasApiURL: String = ""
 
-  @Value("${menas.oozie.splineMongoURL:}")
-  val splineMongoURL: String = ""
+  @Value("${menas.oozie.lineageWriteApiUrl:}")
```

Comment:
As mentioned above, I would get rid of Oozie code to anyhow know about _Spline_ or lineage.